src/chrome/content/rules/ProxyBay.xml

```diff
@@ -0,0 +1,15 @@
+<!--
+	HSTS: max-age=15552000; includeSubDomains; preload || Not in the list yet
```

Comment:
It is: https://dxr.mozilla.org/mozilla-central/source/security/manager/ssl/nsSTSPreloadList.inc#5074